helm/templates/rbac.yaml

```diff
@@ -93,6 +93,20 @@ rules:
   - anarchygovernors
   verbs:
   - '*'
+- apiGroups:
+  - ""
```

Comment:
No need for empty string here. Actually, no need for the empty string for anarchygovernors above either.